boards/x86/qemu_x86/qemu_x86.dts

```diff
@@ -50,6 +50,8 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "intel,pcie";
+		acpi-hw-id = "PNP0A08";
```

Comment:
Should we include dt header and use `PCI_EXPRESS_ROOT_HID_STRING` or similar?

Reply:
it may not same for future platforms

Reply:
>> Should we include dt header and use `PCI_EXPRESS_ROOT_HID_STRING` or similar?
> it may not same for future platforms

Isn't this a standard value though, i.e. I don't think it will change:
https://github.com/acpica/acpica/blob/fee7180f7c1e4cc986440c34a6df559a479421ce/source/common/ahids.c#L283

Reply:
> I suggest it is better to have a common dtsi file say "intel_x86_common.dtsi" and include all nodes which are common for all Intel platforms and then include it in platform specific dtsi files.

That's not a bad idea, although we already have at least a little bit of this kind of "avoid duplication" by having SoC-specific dtsi files which then all boards that use the same SoC include in their dts files.

This is a separate issue from @finikorg's proposal though. Even if we had a common dtsi file for x86 it would still make sense to have defines for the standard PNP IDs (at least the commonly used ones).